controller/deploy/microshift-bootc/config-svc/templates/styles.css

```
@@ -0,0 +1,370 @@
+}
+h2 {
+    color: #333;
+    font-size: 20px;
+    margin-bottom: 15px;
+}
+.section {
+    display: none;
+    padding: 20px 0;
+    animation: fadeIn 0.3s ease-in;
+}
+@keyframes fadeIn {
```

Stylelint (17.4.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)